initial_setup/__init__.py

```diff
@@ -110,6 +118,21 @@ def __init__(self, gui_mode):
         setup_ifcfg_log()
 
     @property
+    def external_reconfig(self):
+        """External reconfig status.
+
+        Reports if external (eq. not triggered by kickstart) has been enabled.
+
+        :returns: True if external reconfig mode has been enabled, else False.
+        :rtype: bool
+        """
+        return self._external_reconfig
```

Comment:
Would it make sense to determine the value here if it's not set? (i.e. set it to ``None`` in ``__init__``)